### PR TITLE
OCPBUGS-14681: fix clock class updates

### PR DESCRIPTION
**Pull request by vitus133**
This commit introduces periodical clock class checks to indicate when there is a change

/cc @aneeshkp @josephdrichard @jzding @nishant-parekh

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -531,39 +581,18 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool) {
...
+						p.pmcCheck = false
+						go p.updateClockClass(&c)
+					}
+
 					if regexErr != nil || !logFilterRegex.MatchString(output) {
 						fmt.Printf("%s\n", output)
 					}
 					out := fmt.Sprintf("%s\n", output)
 					if p.name == ptp4lProcessName {
 						if strings.Contains(output, ClockClassChangeIndicator) {
```
Comment (aneeshkp) [inline pkg/daemon/daemon.go:594]: Do we need this check anymore if we are doing clock class check  in interval ?

Comment (vitus133) [reply pkg/daemon/daemon.go:594]: I thought leaving it here for after ptp4l implements class change notification.
WDYT?

**Pull request merged**